superset-frontend/src/components/ListView/ListView.tsx

```diff
@@ -51,12 +51,15 @@ interface Props {
 const bulkSelectColumnConfig = {
   Cell: ({ row }: any) => (
     <div>
-      <IndeterminateCheckbox {...row.getToggleRowSelectedProps()} />
+      <IndeterminateCheckbox {...row.getToggleRowSelectedProps()} id={row.id} />
```

Comment:
Not the scope of this PR, but have we considered allowing "Shift + Click" to select/unselect all rows between previous and current selected rows? This is a pretty common user interaction expected from a data table with checkboxes.

Another convenient thing to add is to allow users to select a row by clicking anywhere on the row (except links and buttons, of course), instead of just the checkbox.

Reply:
I think the shift+click is something we likely want at some point. I'll see if there' a simple way to do this, otherwise I'll look into adding it in a follow up PR. 

Clicking on the whole row I'm not so sure about. I'll try to get some design feedback